src/pages/BlockDetails.vue

```diff
@@ -157,9 +205,7 @@ const tab = ref<filterTransactionsModel.Tab>('all');
 
 const { width } = useWindowSize();
 
-type HashType = 'full' | 'short'
-
-const hashType = computedEager<HashType>(() =>
+const hashType = computed<'full' | 'medium' | 'short' | 'two-line'>(() =>
```

Comment:
It's better to use `computerEager` here, because the value recomputes frequently, recomputation is cheap, and the result is of primitive type.

Reply:
Acknowledged and committed 👍🏼